frontend/src/pages/LearnPage/LearnPage.tsx

```diff
@@ -115,11 +115,31 @@ const LearnPage: React.FC = () => (
           </ListItem>
           <ListItem>a spoustu dalšího</ListItem>
         </List>
-        <p>Další informace zveřejníme v průběhu srpna.</p>
+        {/* <p>Další informace zveřejníme v průběhu srpna.</p>
```

Comment:
tohle se použije? Jestli ne, tak smazat

Reply:
Nevím. Přijde mi, že je to logické vyhodit. Radši bych zatím nechala a probrala, až to bude k vidění na serveru.

Reply:
Já bych to klidně smazal, když se to to bude vracet, tak je to jednoduchý přidat znova a nebo najít v git historii.

Já jsem zvyklej do masteru nedávat žádný zakomentovaný kód (kromě toho, když vim na 100% že se použije). Protože pak tam ti tam zůstane plno starýho zakomentovanýho kódu co se už nikdy nepoužije.

Reply:
Ok. smazáno;)